scripts/update-appup.sh

```diff
@@ -123,8 +123,12 @@ PREV_REL_DIR="${PREV_DIR_BASE}/${PREV_TAG}/_build/${PROFILE}/lib"
 if [ "${IS_CHECK:-}" = 'yes' ]; then
     diffs="$(git diff --name-only | grep -E '\.appup\.src' || true)"
     if [ "$diffs" != '' ]; then
-        git --no-pager diff
-        echo "$0 ---check produced git diff"
+        git add "$diffs"
+        git commit -m "chore: Update appup"
```

Comment:
Maybe we could only print the warning message, but let the dev inspect and stage the changes manually?

Reply:
This is not going to be committed to the main repo, it's just a quick & dirty way to format a patch.